### PR TITLE
added contributors fetching

### LINKED ISSUE #467: write a script to fetch contributors from github api and fill the required details in the Contributors.md file.

### PR DESCRIPTION
**Pull request by MadKat13**
This pull reques fixes #467

### CODE REVIEW

File: fetch_contributors.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            with urllib.request.urlopen("https://api.github.com/repos/hacktoberfest17/" +
+                                        "programming/contributors?per_page=100&page=" +
+                                        str(PAGE)) as response:
+                HEADERS = response.info()
+                PAGE += 1
+                for contributor in json.loads(response.read()):
+                    CONTRIBUTORS.append((contributor['login'], contributor['html_url']))
+
+#write to the file contributors.md
+FILE = open(os.path.join(os.path.dirname(__file__), 'contributors.md'), "w")
```
Comment (t2013anurag) [inline fetch_contributors.py]: CONTRIBUTORS.md for consistency.

File: fetch_contributors.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Programming contributors (sorted contributions)
+============================================
+
+
+""")
+
+for contributor in CONTRIBUTORS:
+    FILE.write("* **[" + str(contributor[0]) +"]("+str(contributor[1])+")**\n\n\n")
+FILE.close()
+
```
Comment (t2013anurag) [inline fetch_contributors.py]: add a new line, remove space.

File: fetch_contributors.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import urllib.request
+import os
+import json
+
+#List of contributor object (name,weburl)
+CONTRIBUTORS = []
+
+#first page with 100 contributors
+with urllib.request.urlopen("https://api.github.com/repos/hacktoberfest17/" +
```
Comment (t2013anurag) [inline fetch_contributors.py]: make sure we get all the contributors, by default github only shows 100.

Comment (MadKat13) [reply fetch_contributors.py]: I did this with the pagination

**Comment by MadKat13** [comment]
@t2013anurag Added the code corrections

**Pull request merged**